docs/maintaining-a-track.md

```diff
@@ -0,0 +1,79 @@
+# Maintaining an Exercism Language Track
+
+* [Philosophy](#philosophy)
+
+  * [Be Kind](#be-kind)
+  * [Be Responsive](#be-responsive)
+  * [Empower New Contributors](#empower-new-contributors)
+  * [Recognize Great Contributions](#recognize-great-contributions)
+  * [Improve the Learning Experience](#improve-the-learning-experience)
+  * [Collaborate](#collaborate)
+
+* [Day-to-Day Tactics](#day-to-day-tactics)
+
+  * [Watch Key Repositories](#watch-key-repositories)
+  * [Perform Common Tasks](#perform-common-tasks)
+
+## Philosophy
+
+### Be Kind
+
+First and foremost, strive always to be kind, humble and show gratitude for the work being contributed.
```

Comment:
I think both are allowed–English! Never as straight forward as... say... C++ :)

That said, I think you're right that it reads better. I'll change it.